src/Microsoft.Data.SqlClient/netfx/src/Resources/Strings.Designer.cs

```diff
@@ -19,7 +19,7 @@ namespace System {
     // class via a tool like ResGen or Visual Studio.
     // To add or remove a member, edit your .ResX file then rerun ResGen
     // with the /str option, or rebuild your VS project.
-    [global::System.CodeDom.Compiler.GeneratedCodeAttribute("System.Resources.Tools.StronglyTypedResourceBuilder", "15.0.0.0")]
+    [global::System.CodeDom.Compiler.GeneratedCodeAttribute("System.Resources.Tools.StronglyTypedResourceBuilder", "16.0.0.0")]
```

Comment:
Let's keep this unchanged, unless needed.

Reply:
It was updated automatically when I modified `Strings.resx`. Should I change them back manually?

Reply:
@David-Engel what you suggest?

Reply:
I would defer on the side of reverting these "automatic" changes by tooling unless they can explained in context of this PR.

Reply:
@ErikEJ answering here,

No, We don't want to enforce VS 2019 as of now. I was wondering if this automatic change means something else, otherwise we can revert it.